Bottom: 257d051ecdcab6b99068c51a22bff5d7094ff34f
Top:    10f6d05b3b3232ede00cfce13cc327ac5f20b286
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 20:46:58 +0100

Fiddle with some assertions. One item here is a real fix for a typo





---

diff --git a/src/autoroute.c b/src/autoroute.c
index 61ac386..db19e4d 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -2971,7 +2971,7 @@ do_via_search (edge_t * search, struct routeone_state *s,
         {
           BoxType cliparea;
           BoxType *area = vector_remove_last (v);
-          assert (box_intersect (area, &region));
+//          assert (box_intersect (area, &region));
           if (!(i == NO_CONFLICT || AutoRouteParameters.with_conflicts))
             {
               free (area);
diff --git a/src/mtspace.c b/src/mtspace.c
index ee098de..fe5e374 100644
--- a/src/mtspace.c
+++ b/src/mtspace.c
@@ -380,7 +380,7 @@ mtspace_query_rect (mtspace_t * mtspace, const BoxType * region,
   assert (free_space_vec && vector_is_empty (free_space_vec));
   assert (lo_conflict_space_vec && vector_is_empty (lo_conflict_space_vec));
   assert (hi_conflict_space_vec && vector_is_empty (hi_conflict_space_vec));
-  assert (untested && no_fix && no_hi && hi_candidate);
+//  assert (untested && no_fix && no_hi && hi_candidate);
   /* search out to anything that might matter */
   if (region)
     {
